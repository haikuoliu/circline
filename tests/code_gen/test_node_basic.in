node a = node(1);
print(a);

a = node(1.2);
print(a);

a = node("Hello World!");
print(a);

a = node(1<2);
print(a);

node f(int a) {
	return node(a);
}

print(f(22));