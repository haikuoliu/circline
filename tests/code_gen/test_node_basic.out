node:1
node:1.200000
node:Hello World!
node:true
node:22
